src/index.js

```diff
@@ -259,6 +259,7 @@ module.exports = (
 
   function finalizeHandler () {
     const assets = Object.create(null);
```

Comment:
I'm curious, why use an object instead of Map?

Reply:
I guess I'm just used to exposing API output as JSON-like data?